package/Config.in

```diff
@@ -2704,6 +2704,9 @@ menu "WebOS"
 	source "package/libwebosi18n/Config.in"
 	source "package/luna-service2/Config.in"
 	source "package/umediaserver/Config.in"
+	source "package/webos-gmp/Config.in"
+	source "package/webos-gnutls/Config.in"
+	source "package/webos-userland/Config.in"
```

Comment:
Duplicate webos-userland include?

Reply:
Oops! Happened during rebasing from #46 to current HEAD.